test/storage/sql_table_test.cpp

```diff
@@ -526,9 +544,10 @@ TEST_F(SqlTableTests, ScanTest) {
 
   // manually set the version of the transaction to be 1
   table.version_ = storage::layout_version_t(1);
-  table.AddColumn(txn, "new_col", type::TypeId::INTEGER, true, catalog::col_oid_t(2));
+  table.AddColumn(txn, "new_col", type::TypeId::INTEGER, true, catalog::col_oid_t(2),
+                  table.intToByteArray(new_col_default_value));
 
-  // insert (300, 10002, null)
+  // insert (300, 10002, 1729)
   table.StartInsertRow();
   table.SetIntColInRow(catalog::col_oid_t(0), 300);
   table.SetIntColInRow(catalog::col_oid_t(1), datname_map[300]);
```

Comment:
Column with default value added before this insertion started.  I'm pretty sure the expectation is that the query plan would handle setting the default value, not us.